src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/on_system_request_notification.cc

```diff
@@ -132,6 +132,8 @@ void OnSystemRequestNotification::Run() {
       (*message_)[strings::msg_params][strings::timeout] =
           policy_handler.TimeoutExchangeSec();
     }
+  } else if (file_system::ReadBinaryFile(file_name, binary_data) == true) {
```

Comment:
@ZhdanovP `if (bool func())`. Why do you use bool func() == true ?

Reply:
@AByzhynar please take a look f2dd4a6562711fbaaf71ad9b7cb53d9ea4c81c0e